### PR TITLE
Basic UI

### PR DESCRIPTION
**Pull request by benshidanny11**
# In this milestone I:
- Implemented basic UI (Command line interface)
- Created the app file that contains all create and list data features
- Created the main file that works as the entry file for the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    end
+    run
+  end
+
+  def person_list
+    @persons.each do |individual|
+      puts "[#{individual.class}] Name: #{individual.name}, Age: #{individual.age}"
+    end
+    run
+  end
```
Comment (Mre55) [inline app.rb]: When listing all people the console order is not correct it consoles age to Name and vice versa.

image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    @persons << Teacher.new(specialization, name, age)
+    puts 'create teacher'
+  end
```
Comment (Mre55) [inline app.rb]: Kindly, fix creating teacher method it's crashing the app.

image

Here I attached the screenshot video where using the app and some of its features are crashing the app.

https://user-images.githubusercontent.com/34578631/180096683-38a88dba-f036-4fb3-95db-1cdd2b702ad6.mov

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+      print('Date: ')
+      selected_date = gets.chomp.to_s
+      puts
+
+      handled_rental(selected_book, selected_person, selected_date)
+    else
+      puts 'No books or no persons yet!'
+      run
+    end
+  end
```
Comment (Mre55) [inline app.rb]: Creating rental is also crashing the app. Kindly, this features.

image

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @benshidanny11,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (rloterh) [reply app.rb]: This issue is well resolved! ✔️

File: app.rb (hunk 2 above)
Comment (rloterh) [reply app.rb]: This issue is well resolved! ✔️

File: app.rb (hunk 3 above)
Comment (rloterh) [reply app.rb]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Daniel,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉🌟 

## Highlights 👍
- Interactive console apps properly created

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benshidanny11** [comment]
@rloterh Thanks for approving my project

**Pull request merged**